var/spack/repos/builtin/packages/datatransferkit/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Datatransferkit(CMakePackage):
+    """DataTransferKit is an open-source software library of
+    parallel solution transfer services for multiphysics simulations"""
+
+    homepage = "datatransferkit.readthedoc.io"
+    url      = "https://github.com/ORNL-CEES/DataTransferKit/archive/3.1-rc1.tar.gz"
+    git      = "https://github.com/ORNL-CEES/DataTransferKit.git"
+
+    version('master', branch='master', submodules=True)
+
```

Comment:
Typically, you would want to add at least one version with shasum.

Reply:
Not here, because there are no appropriate release

Reply:
But you do have `3.1-rc1` release/tag?

Reply:
That's 9 months old. It's way too old.

Reply:
Maybe it's time to cut a new release then? 9 months old is pretty new for scientific software, someone might want to use that version.

Reply:
Maybe it's time to cut a new release then? 9 months old is pretty new for scientific software, someone might want to use that version.

Reply:
We release once a year, I would like to keep it that way.

Reply:
In that case, can we add the latest release and install that by default?

Reply:
The latest release is already part of spack as a variant of Trilinos. What we have done is rework the build system so instead of being a variant of Trilinos, we now use Trilinos as a TPL. Because the new build system is incompatible with the old one, we cannot use the previous release. If it is necessary to have a release, I can just tag the current master commit with a dummy release number.

Reply:
Ah, a build system change is a good reason then, I think this is fine as is.